docs/conf.py

```diff
@@ -158,8 +158,8 @@
     # "twitter_url": "https://twitter.com/PyData",
     "logo": {
         "text": "PyData Theme",
+        "alt_text": "",  # PST logo is decorative so mute the alt text
```

Comment:
My first attempt to fix the Axe test failure was to comment this line out, but the theme puts "Logo image" as the default alt text if the client does not specify alt text, which resulted in the home page link appearing as "Logo image PyData Theme" to assistive tech.

Reply:
once #1472 is merged then we *could* simply comment out / remove this line though, right?  i.e. #1472 would make this PR moot?

Reply:
scratch that: not moot, but possible to simply delete the line.

Reply:
👍